drivers/pinctrl/pinctrl_gd32_af.c

```diff
@@ -25,7 +25,7 @@ BUILD_ASSERT((GD32_OSPEED_2MHZ == GPIO_OSPEED_2MHZ) &&
 #else
 	     (GD32_OSPEED_25MHZ == GPIO_OSPEED_25MHZ) &&
 	     (GD32_OSPEED_50MHZ == GPIO_OSPEED_50MHZ) &&
-	     (GD32_OSPEED_200MHZ == GPIO_OSPEED_200MHZ) &&
+	     (GD32_OSPEED_200MHZ == GPIO_OSPEED_MAX) &&
```

Comment:
I think we should update at Zephyr side from GD32_OSPEED_200MHZ to GD32_OSPEED_MAX too. The main reason is that there is no more mention in hal_gigadevice and keep as is will create an inconsistence at Zephyr side.

Reply:
Yes indeed.

Reply:
good point